network/core-node/admin-guide/prerequisites.mdx

```diff
@@ -3,32 +3,79 @@ title: Prerequisites
+### Inbound
+
+A Stellar Core node needs to allow all IPs to connect to its `PEER_PORT` over TCP. You can specify a port when you [configure] Stellar Core, but most people use the default, which is **11625**.
+
+### Outbound
+
+A Stellar Core node needs to connect to other nodes on the internet via their `PEER_PORT` over TCP. You can find information about other nodes' `PEER_PORT`s on a network explorer like [Stellarbeat](https://stellarbeat.io/), but most use the default port for this as well, which is (again) **11625**.
 
 ## Internal System Access
 
-Stellar Core also needs to connect to certain internal systems, though exactly how varies based on your setup.
+Stellar Core also needs to connect to certain internal systems, though exactly how this is accomplished can vary based on your setup.
+
+### Inbound
+
+- Stellar Core exposes an _unauthenticated_ HTTP endpoint on its `HTTP_PORT`. You can specify a port when you [configure] Stellar Core, but most people use the default, which is **11626**.
+- The `HTTP_PORT` is used by other systems (such as Horizon) to submit transactions, so this port may have to be exposed to the rest of your internal IP addresses.
+- It's also used to query Stellar Core [info](./commands.mdx) and provide [metrics](./monitoring.mdx).
+- And to perform administrative commands such as [scheduling upgrades](./network-upgrades.mdx) and changing log levels
+- For more on that, see [commands](./commands.mdx)
+
+:::note Note on exposing the HTTP endpoint
+
+If you need to expose this endpoint to other hosts in your local network, we strongly recommended you use an intermediate reverse proxy server to implement authentication. Don't expose the HTTP endpoint to the raw and cruel open internet.
+
+:::
+
+### Outbound
+
+- Stellar Core requires access to a database (PostgreSQL for example). If that database resides on a different machine on your network, you'll need to allow that connection. You'll specify the database when you [configure] Stellar Core.
+- You can safely block all other connections.
+
+## Storage
+
+Stellar Core stores two copies of the ledger: one in a SQL [database](#database) and one in XDR files on local disk called [buckets](#buckets). For the most part, the contents of both the database and buckets directories can be ignored, since they are completely managed by Stellar Core. In terms of storage space, 100 GB is enough in 2024. Most people seem to have at least 1TB on hand.
+
+### Database
+
+The database is consulted during consensus, and modified atomically when a transaction set is applied to the ledger. It's random access, fine-grained, and fast.
+
+While a SQLite database works with Stellar Core, we generally recommend using a separate PostgreSQL server. A Postgres database is the bread and butter of Stellar Core.
```

Comment:
Just want to verify: does SQLite still work with Stellar Core?

Reply:
i think SQLite _can_ work? I think I remember using it recently when I was working on something soroban-rpc-related. perhaps it's just captive core that can/should use sqlite?

Reply:
I believe it.